app/views/support_interface/application_forms/show.html.erb

```diff
@@ -62,25 +62,33 @@
   <% end %>
 <% end %>
 
-<h2 class="govuk-heading-l govuk-!-margin-top-8" id="qualifications">Qualifications</h2>
 
-<%= render QualificationsComponent.new(application_form: @application_form) %>
-
-<h2 class="govuk-heading-l govuk-!-margin-top-8">Work history</h2>
-
-<%= render WorkHistoryReviewComponent.new(application_form: @application_form, editable: false, heading_level: 3) %>
-
-<h2 class="govuk-heading-l govuk-!-margin-top-8">Unpaid experience and volunteering</h2>
-
-<%= render VolunteeringReviewComponent.new(application_form: @application_form, editable: false, heading_level: 3) %>
-
-<h2 class="govuk-heading-l govuk-!-margin-top-8">Language skills</h2>
-
-<%= render LanguageSkillsComponent.new(application_form: @application_form) %>
-
-<h2 class="govuk-heading-l govuk-!-margin-top-8">Personal statement</h2>
-
-<%= render PersonalStatementComponent.new(application_form: @application_form) %>
+<h2 class="govuk-heading-l govuk-!-margin-top-8">Application data</h2>
+
+<div class="govuk-tabs" data-module="govuk-tabs">
+  <h2 class="govuk-tabs__title">
+    Contents
+  </h2>
+  <ul class="govuk-tabs__list">
+    <li class="govuk-tabs__list-item">
+      <a class="govuk-tabs__tab" href="#what-the-provider-sees">
+        What the provider sees
+      </a>
+    </li>
+    <li class="govuk-tabs__list-item">
+      <a class="govuk-tabs__tab" href="#what-the-candidate-sees">
+        What the candidate sees
+      </a>
+    </li>
+  </ul>
+
+  <div class="govuk-tabs__panel govuk-tabs__panel--hidden" id="what-the-provider-sees">
```

Comment:
```suggestion
  <div class="govuk-tabs__panel" id="what-the-provider-sees">
```

Reply:
I think the selected panel is not meant to be hidden in the initial server-side render?

Reply:
I got this from the Design System: https://design-system.service.gov.uk/components/tabs/

Reply:
Seems legit then, `--hidden` is probably unused without `.js-enabled` on body